core/src/test/java/org/elasticsearch/bootstrap/BootstrapCheckTests.java

```diff
@@ -197,4 +199,15 @@ public void testEnforceLimits() {
         assertTrue(BootstrapCheck.enforceLimits(settings));
     }
 
+    public void testMinMasterNodes() {
+        BootstrapCheck.Check check = new BootstrapCheck.MinMasterNodesCheck();
+        assertTrue(check.check(Settings.EMPTY));
+        assertFalse(check.check(
+            Settings.builder().put(ElectMasterService.DISCOVERY_ZEN_MINIMUM_MASTER_NODES_SETTING.getKey(), 1).build()
+        ));
+        expectThrows(RuntimeException.class, () ->
+            BootstrapCheck.check(
```

Comment:
I wanted to check that the default set of checks enforces this. shall I add BoostrapCheck.check(boolean, settings)?